tasks/publish-to-mrrc/publish-to-mrrc.yaml

```diff
@@ -0,0 +1,59 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: publish-to-mrrc
+  labels:
+    app.kubernetes.io/version: "0.1.0"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  params:
```

Comment:
The `cosignPubKeyConfig` parameter is missing? It is mentioned in the README. But now I see that the value is hardcoded under `volumes`. I would recommend adding the parameter with a default value of what you have - I think it's nicer than hardcoding it inside the task. WDYT?

And now I see that we had a discussion related to this here: https://github.com/konflux-ci/release-service-catalog/pull/510/files#r1717348622

Reply:
Removed in new commit.